data/json/items/armor/holster.json

```
@@ -190,8 +190,8 @@
         "pocket_type": "CONTAINER",
         "holster": true,
         "min_item_volume": "1250 ml",
-        "max_contains_volume": "3750 ml",
-        "max_contains_weight": "2 kg",
+        "max_contains_volume": "8000 ml",
+        "max_contains_weight": "8.2 kg",
```

```suggestion
        "max_contains_weight": "8200 g",
```
units can't take decimals.